keyboards/finalkey/origin65a/info.json

```diff
@@ -0,0 +1,233 @@
+{
+    "keyboard_name": "Origin65A",
+    "manufacturer": "Finalkey",
+    "maintainer": "Finalkey",
+    "usb": {
+        "vid": "0xECED",
+        "pid": "0x3663",
+        "device_version": "0.0.1"
+    },
+    "debounce": 5,
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["F5", "F4", "F1", "F0", "E6", "B1", "B2", "B3", "D4", "D6", "D7", "B4", "B5", "B6", "C7"],
+        "rows": ["D0", "D1", "D2", "D3", "D5"]
+    },
+    "processor": "atmega32u4",
+    "bootloader": "caterina",
```

Comment:
This uses a Pro Micro? If not, this should be `atmel-dfu`.

Reply:
i don't know why, but the PCB come with keyboard was using Caterina bootloader.

Reply:
No worries then, as long as you're aware and it's not just some copy pasta.

Reply:
Yeah, it kinda annoying. But the bootloader was modify and decrease timeout only 4 seconds, not like original was 8 seconds.